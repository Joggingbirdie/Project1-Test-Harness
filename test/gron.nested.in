{"person": {"name": "Alice", "address": {"city": "London", "zipcode": "SW1A"}}}
