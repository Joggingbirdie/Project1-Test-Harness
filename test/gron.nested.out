json = {};
json.person = {};
json.person.name = "Alice";
json.person.address = {};
json.person.address.city = "London";
json.person.address.zipcode = "SW1A";
